apns/analysis/apns2_eta_abacus.py

```diff
@@ -0,0 +1,304 @@
+"""Band structure similarity test for ABACUS"""
+
+def collect_jobs(folder: str):
+    """Collect APNS jobs for band structure similarity calculation"""
+    import os, json, re
```

Comment:
thanks, you can change on my code directly, but that may make the code looks not quite nice after change because the length of line may lose balance, if you look at next following lines